runtime/lua/vim/lsp.lua

```diff
@@ -1214,6 +1218,25 @@ function lsp._with_extend(name, options, user_config)
   return resulting_config
 end
 
+--- @param handler? lsp.Handler
+--- @param silent? boolean
+--- @return lsp.Handler
```

Comment:
Shouldn't this be `lsp.Handler | nil`?

Reply:
No because a function is always returned.